CHANGELOG.md

```diff
@@ -1,7 +1,288 @@
+- `alpaka::Vec`:
+  - Added generator constructor #2085
+  - Added `front` and `back` methods #2085
+  - Added `elementwise_{min,max}` methods #1805
+  - `Vec` now features a deduction guide for easier construction #1610
+- Documentation:
+  - Added example illustrating typical data-parallel patterns with alpaka #1712
+  - Added documentation about the behaviour of `constexpr` functions in kernel code #1699
+  - Added documentation about CUDA function attributes #1697
+  - Added documentation about setting the C++ standard library for clang #1695
+- Test cases:
+  - Added test for `alpaka::ViewSubView` #2095
+  - Added queue test which checks that a task is destroyed after execution #2047
+  - Added test for `alpaka::getValidWorkDiv` with `Idx` type #1830
+  - Added tests for `alpaka::subDivideGridElements` #1829
+- CI:
+  - Run test cases with `-Werror` #2163
+  - Added UBSan CI job #2059
+  - Added CI job to create amalgamated `alpaka.hpp` #1956 #1965 #1972
+  - Made GitLab CI jobs interruptible #1904
+  - Updated used Boost and CMake versions #1903 #1969
+  - Added `agc-manager` support #1871 #1921
+  - Added TSan CI job #1851 #2103 #2137
+  - GitLab CI jobs are now automatically generated #1785 #1889 #1896 #1951 #1952 #2005 #2041
+- Upgraded to `clang-format-16` #2147
+- Added `alpaka::getPitchesInBytes` function which returns all pitches for a given view as an `alpaka::Vec` #2092 #2093 #2116 #2125
+- Added `alpaka::get{Extents,Offsets}` functions which return all extents/offsets for a given view as an `alpaka::Vec` #2080
+- Added `alpaka_DISABLE_VENDOR_RNG` CMake flag and its corresponding preprocessor macro `ALPAKA_DISABLE_VENDOR_RNG` to optionally disable vendor RNG libraries #2036
+- Added alpaka port of BabelStream #1846 #1934
+- Added utility functions `alpaka::core::{divCeil,intPow,nthRootFloor}` #1830
+- Added `operator==(alpaka::WorkDivMembers const&, alpaka::WorkDivMembers const&)` as a `friend` of `alpaka::WorkDivMembers` #1829
+- Added `alpaka::is{Accelerator,Device,Platform,Queue}` template variables #1818
+- Added accelerator tags which allow for accelerator-specific code paths without enabling the corresponding back-end #1804 #1814
+- Added support for `std::mdspan` #1788 #2048 #2052 #2053
+- Added `alpaka::ViewConst` which wraps another view but only allows non-modifying accesses #1746
+- `alpaka::{memcpy,memset}` now support temporary destination views #1743
+- Host memory alignment can now be specified by using the `ALPAKA_DEFAULT_HOST_MEMORY_ALIGNMENT` macro #1686
+- Added `alpaka::allocMappedBuf` for allocating device-accessible pinned host memory #1685 #1782 #2162
+  - Added related trait `alpaka::trait::hasMappedBufSupport` to query the host CPU for device-accessible pinned memory support #1782
+  - Added related utility function `alpaka::allocMappedBufIfSupported` to allocate device-accessible pinned memory if supported and regular memory otherwise #1782 #2120
```

Comment:
```suggestion
  - Added related utility function `alpaka::allocMappedBufIfSupported` to allocate device-accessible pinned memory, if supported, and regular memory otherwise #1782 #2120
```